plugins/modules/aws_msk_config.py

```diff
@@ -14,7 +14,6 @@
 version_added: "2.0.0"
 requirements:
     - botocore >= 1.17.48
```

Comment:
Not sure about this, but since we are requiring `botocore >= 1.19.0`, I guess we can also remove this (if there is a specific option requiring `botocore < 1.19.0`, not sure how we do handle that)

Reply:
> I guess we can also remove this (if there is a specific option requiring botocore < 1.19.0, not sure how we do handle that)

I don't see that the integrationtest installs a specific version. So basically CI must fail if it's requiring a lower botocore version.  
I'm going to remove it also.